app/views/partials/_recovery_code.html.slim

```diff
@@ -1,5 +1,5 @@
 #recovery-code(class="col-12 border-box mt4 mb3 py2 px3 fs-20p sans-serif\
```

Comment:
I think if we update this outer container to a `px4` and remove the `ml2` on the container around the words the left and right edges of the content will mimic ryan's design a little closer.